chromium/rules.js

```diff
@@ -3,10 +3,21 @@
 (function(exports) {
 
 // Stubs so this runs under nodejs. They get overwritten later by util.js
-var DBUG = 2;
-var INFO = 3;
-var WARN = 5;
-function log(){}
+if (typeof util == 'undefined') {
```

Comment:
Suggestion: move `util.js` in manifest to the top, so that in extension you would always have global `util` already, and in the Node.js you wouldn't. Then, this entire block can be changed to just `var util = typeof util !== 'undefined' ? util : { log: () => {} };` (such typeof check is valid because `var` declarations get merged with globals).

Reply:
Good  point about moving utils.js, will do.

However, this code is in an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression) so a variable declared here with `var ...` should not be added to the global scope. I'm pretty sure it should still be added manually to `window`.